pageserver/src/tenant/config.rs

```diff
@@ -410,9 +405,7 @@ impl TenantConfOpt {
             image_layer_creation_check_threshold: self
                 .image_layer_creation_check_threshold
                 .unwrap_or(global_conf.image_layer_creation_check_threshold),
-            switch_aux_file_policy: self
-                .switch_aux_file_policy
-                .unwrap_or(global_conf.switch_aux_file_policy),
+            switch_aux_file_policy: None,
```

Comment:
Why do we have to keep this? Tenant configs are persisted on disk, but could we just ignore the field when de-serializing?